Source/items.cpp

```diff
@@ -1313,8 +1313,12 @@ void GetStaffSpell(int i, int lvl, BOOL onlygood)
 	int l, rv, s, minc, maxc, v, bs;
 	char istr[64];
 
+#ifdef HELLFIRE
+	if (0) {
+#else
 	if (!random(17, 4)) {
 		GetItemPower(i, lvl >> 1, lvl, 256, onlygood);
+#endif
 	} else {

```

Comment:
```suggestion
#ifndef HELLFIRE
	if (!random(17, 4)) {
		GetItemPower(i, lvl >> 1, lvl, 256, onlygood);
	} else
#endif
	{
```

Reply:
Github multiline comments doesn't combine with suggestions :( You will have to copy this your self.

Reply:
applied